admin_manual/release_notes.rst

```diff
@@ -12,6 +13,61 @@ Release Notes
 * :ref:`8.0_release_notes_label`
 * :ref:`7.0_release_notes_label`
 
+.. _10.0.4_release_notes_label:
+
+Changes in 10.0.4
+-----------------
+
+Dear ownCloud administrator, please find below the changes and known issues of ownCloud Server 10.0.4 that need your attention:
+
+**The full ownCloud Server 10.0.4 changelog can be found here: https://github.com/owncloud/core/blob/stable10/CHANGELOG.md**
+
+* **More granular sharing restrictions**
+The option "Restrict users to only share with users in their groups" in the admin sharing settings forces users to only be able to share with groups they are member of while simultaneously prohibiting to share with single users that do not belong to any of the users' groups. To make this more granular we split this option into two and added "Restrict users to only share with groups they are member of" which now differentiates between users and groups making it possible to force users to be able to share with all users of an installation but only with groups they are member of and vice versa. Please check if the new possibilities better suit your needs.
+
+- **Configurable solution for indistinguishable user display names**
+In the ownCloud sharing dialog users are displayed according to their display name. As users can choose their display name in self-service (can be disabled in config.php) and display names are not unique it is possible that a user can't distinguish sharing results. To cover this case the displayed user identifiers are now configurable. Administrators can configure to additionally display mail addresses or user ids. Please check if this new feature better suits your needs.
```

Comment:
How about the following:

---
The ownCloud sharing dialog displays users according to their display name. As users can choose their display name in self-service (which can be disabled in `config.php`) and display names are not unique, it is possible that a user can't distinguish sharing results. To cover this case the displayed user identifiers are now configurable. Administrators can now configure the display of either mail addresses or user ids. Please check if this new feature better suits your needs.
---